.env.default

```diff
@@ -0,0 +1,2 @@
+NEXMO_API_KEY=
```

Comment:
I believe these should be `OPENTOK_API_KEY`, etc.

Reply:
Fixed in later commit